SourceX/display.cpp

```diff
@@ -121,8 +123,14 @@ bool SpawnWindow(const char *lpWindowName)
 #ifdef USE_SDL1
 	SDL_EnableUNICODE(1);
 #endif
-#if defined(USE_SDL1) || defined(__SWITCH__)
-	InitController();
+#ifdef USE_SDL1
+	// On SDL 1, there are no ADDED/REMOVED events.
+	// Always try to initialize the first joystick.
+	Joystick::Add(0);
+#ifdef __SWITCH__
+	// TODO: Is this really needed on the Switch?

```

Comment:
I was at the time, but I don't know if the SDL port has fixed it by now.

Reply:
Yeah, this is something to investigate for someone with a hacked Switch
This PR preserves the previous behaviour

Reply:
```suggestion
	// TODO: There is a bug in SDL2 on Switch where it does not repport controllers on startup (Jan 1, 2020)
```